components/GitHubCallout/GitHubCallout.mdx

```diff
@@ -0,0 +1,44 @@
+export const GitHubCallout = ({ type, children }) => {
+  let color;
+  let icon;
+  switch (type) {
+    case 'tip':
+      color = 'green';
+      icon = 'fa-lightbulb';
+     break;
+    case 'important':
+      color = 'purple';
+      icon = 'fa-message-exclamation';
+     break;
+    case 'warning':
+      color = 'yellow';
+      icon = 'fa-triangle-exclamation';
+     break;
+    case 'caution':
+      color = 'red';
+      icon = 'fa-stop';
+     break;
+    case 'note':
+    default:
+      color = 'blue';
+      icon = 'fa-circle-info';
+    break;
+  }
+
+  return (
+		<div className={`pl-3 border-l-1 border-${color}-500 dark:border-${color}-400`}>
```

Comment:
This may or may not work, just make sure it does! (I can see it working in the preview but not working when used)

The reason is that Tailwind compiles down the strings for performance reasons, so it doesn't understand interpolation like this.

If it doesn't work, the way to fix it is to do this:

```
color  = 'border-green-500';
...
<div className={`pl-3 border-l-1 ${color}`}>
```

(Tailwind just uses a regex to find everything that fits, so as long as it's written SOMEWHERE in the code it'll catch it. But it can't deal with variables like this.)

(If it works in the preview, it should work everywhere! If not that's a bigger issue we should address.)

Reply:
(If you get stuck here, just ping me and I can help out! Both for this PR and ReadMe in general)

Reply:
My eyes may be deceiving because of the minor difference I put in for the light vs dark colors but it _seems_ to be working as-is?

![Screenshot 2025-04-01 at 4 35 45 PM](https://github.com/user-attachments/assets/86ee33f1-ba86-4809-bbf4-468d87d50e8a)
![Screenshot 2025-04-01 at 4 35 49 PM](https://github.com/user-attachments/assets/2f9b316e-d404-48d4-aff8-dc10c1fce25c)